schemas/author.json

```diff
@@ -85,13 +98,25 @@
 			"type": "string",
 			"description": "An optional URL to this contributor's webpage.",
 			"validation": [
-				"url"
+				"url",
+				"nullable"
 			]
 		},
 		"userGroupId": {
 			"type": "integer",
 			"description": "in which user group should this contributor be identified. Usually one of the author roles, such as Author or Translator.",
```

Comment:
The ID of this contributor's assigned user group. See `userGroupName`.

Reply:
changed